src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -100,8 +115,78 @@ class AmendmentTableImpl : public AmendmentTable
 void
 AmendmentTableImpl::addInitial ()
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+    std::set<std::pair<uint256, std::string>> toAdd;
+
+    for (auto const& _ : detail::PreEnabledAmendments)
```

Comment:
I really don't believe that `_` is a valid name for development.  From the C++ standard, 

> Each name that begins with an underscore is reserved to the implementation for use as a name in the global namespace.

There is a later rule about names that start `_` and a capital letter but the rule above seems unambiguous.